tests/checks/interceptor_otel_tracing/interceptor_otel_tracing_test.go

```diff
@@ -257,8 +258,8 @@ func TestTraceGeneration(t *testing.T) {
 	// Send a test request to the interceptor
 	sendLoad(t, kc, data)
 
-	// setting sleep for 5 sec so traces are sent over
-	time.Sleep(5 * time.Second)
+	// setting sleep for 15 sec so traces are sent over
+	time.Sleep(15 * time.Second)
```

Comment:
I will try to see if we can replace this with something more deterministic than sleep, but go ahead and merge as is for no